examples/source/components/amp-recaptcha-input.html

```diff
@@ -14,7 +13,7 @@
 <head>
   <meta charset="utf-8">
   <title>amp-recaptcha-input</title>
-  <link rel="canonical" href="<%host%>/components/amp-recaptcha-input/">
+  <link rel="canonical" href="s/host/hosts.platform/components/amp-recaptcha-input/">
```

Comment:
sorry for not being clear enough, this needs to be:

```
"<%host.platform%>/components/amp-recaptcha-input/"
```

Sidenote: I was using the old fashioned sed way to describe a change, the syntax means substitute OLD with NEW `s/OLD/NEW/`.